src/metrics.cpp

```diff
@@ -225,12 +268,25 @@ int printStats(bool mining)
 
     if (IsInitialBlockDownload(Params())) {
         int netheight = currentHeadersHeight == -1 || currentHeadersTime == 0 ? 
-            0 : EstimateNetHeight(Params().GetConsensus(), currentHeadersHeight, currentHeadersTime);
+            0 : EstimateNetHeight(params, currentHeadersHeight, currentHeadersTime);
         int downloadPercent = height * 100 / netheight;
         std::cout << "     " << _("Downloading blocks") << " | " << height << " / ~" << netheight << " (" << downloadPercent << "%)" << std::endl;
```

Comment:
The way spacing is done here does not really account for localisation. This is a pre-existing problem that does not block this PR.

Reply:
We maybe need to open an issue for this too unless there is one already.